Formula/symengine.rb

```diff
@@ -0,0 +1,52 @@
+class Symengine < Formula
+  desc "Fast symbolic manipulation library written in C++"
+  homepage "https://sympy.org"
+  url "https://github.com/symengine/symengine/releases/download/v0.8.1/symengine-0.8.1.tar.gz"
+  sha256 "41eb6ae6901c09e53d7f61f0758f9201e81fc534bfeecd4b2bd4b4e6f6768693"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "flint"
+  depends_on "gmp"
+  depends_on "libmpc"
+  depends_on "llvm"
+  depends_on "mpfr"
+
+  def install
```

Comment:
```suggestion
  depends_on "mpfr"

  fails_with gcc: "5"

  def install
```
This should help with the Linux build failure.